sdk/python/feast/ui_server.py

```diff
@@ -51,7 +51,7 @@ def shutdown_event():
 
     async_refresh()
 
-    ui_dir_ref = importlib_resources.files(__name__) / "ui/build/"
+    ui_dir_ref = importlib_resources.files(__spec__.parent) / "ui/build/"  # type: ignore[name-defined]
```

Comment:
Type ignore due to a mypy issue:
https://github.com/python/mypy/pull/14739

It was merged 2 weeks ago, but we're using `mypy` 1.10 which is latest currently, and the change might not have been released.